crypto/keys/types.go

```diff
@@ -46,9 +46,11 @@ type Keybase interface {
 	// The following operations will *only* work on locally-stored keys
 	Update(name, oldpass string, getNewpass func() (string, error)) error
 	Import(name string, armor string) (err error)
+	ImportPrivKey(name string, armor string, passphrase string) error
```

Comment:
```suggestion
	ImportPrivKey(name, armor, passphrase string) error
```

Reply:
I'm most definitely fine with this - will adjust the style of the whole file in an ensuing PR